packages/storefront-lwc/src/modules/commerce/productDetail/productDetail.js

```diff
@@ -9,6 +9,7 @@ import { ShoppingCart } from 'commerce/data';
 import { canAddToCart } from './product.helper.js';
 import { useQuery } from '@lwce/apollo-client';
 import gql from 'graphql-tag';
+import { apiClient } from '../api/client';
```

Comment:
apiClient will be unused. Hopefully `import '../api/client';` should give us the side effect of setClient() as expected.